spec/release_spec.rb

```diff
@@ -69,15 +69,15 @@ module Fauxpaas
       let(:remote_runner) { double(:remote_runner, run: double(:status, success?: true)) }
       let(:command) { "somecommand" }
       let(:sites) do
-        Sites.new({
+        Sites.new(
           "site1" => ["host1", "host2"],
           "site2" => ["host3", "host4"]
-        })
+        )
       end
 
       describe "#run_per_host" do
         it "runs the command at each host" do
-          %w{host1 host2 host3 host4}.each do |hostname|
+          ["host1", "host2", "host3", "host4"].each do |hostname|
```

Comment:
I just adjusted prefs elsewhere for this rule. I like the %w for ergonomics, but I'm not offended if a short array uses commas. I definitely prefer square brackets to the braces though.

https://github.com/mlibrary/ae-utils/blob/ce620983d66363d96e842d9de35f965f562ce7cc/.rubocop.yml#L106

Reply:
If the only magic array literal that existed were this one, I'd be happy with it. However, there's half a dozen, and they can be very surprising.  I don't want to change the rule right at this time.